src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,66 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, S> {
+    private T integer;
+    private S string;
```

Comment:
```suggestion
    private T first;
    private S second;
```
let's rename the fields - what if we will parameterise the Pair object with different types that Integer and String?